src/Release/CreateRelease.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Clearbooks\Labs\Release;
+use Clearbooks\Labs\Release\Gateway\ReleaseGateway;
+use Clearbooks\Labs\Release\UseCase\CreateRelease\Request;
+
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 15/07/15
+ */
+class CreateRelease
+{
+    /**
+     * @var ReleaseGateway
+     */
+    private $releaseGateway;
+
+    /**
+     * @param ReleaseGateway $releaseGateway
+     */
+    function __construct( ReleaseGateway $releaseGateway )
+    {
+        $this->releaseGateway = $releaseGateway;
+    }
+
+
+    /**
+     * @param Request $request
+     * @return int|bool
```

Comment:
Hmm I'm not so sure about returning an int or a bool.

What does the bool mean?

Perhaps return an interface:

```php
interface Response {
    /** @return bool */
    public function isSuccessful();
    
    /* @return string[] */
    public function getValidationErrors();

    /** @return string */
    public function getReleaseId();
}
```